edelphi/src/main/java/fi/metatavu/edelphi/languagedetect/TextLanguageDetector.java

```diff
@@ -0,0 +1,64 @@
+package fi.metatavu.edelphi.languagedetect;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.api.client.repackaged.com.google.common.base.Optional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.api.client.repackaged.com.google.common.base.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)